### ISSUE TITLE
Missing LayerNormalization & MultiHeadAttention Keras Layers

### ISSUE THREAD

**Issue by lqdev** [SciSharp/TensorFlow.NET#797]
@Oceania2018 can you assign this issue to me. I'll try and take a stab at it.

I was trying to see whether implementing the [Keras Transformer recommendation sample](https://keras.io/examples/structured_data/movielens_recommendations_transformers/#create-a-bst-model) is feasible in .NET. However, I believe two of the layers, [LayerNormalization](https://github.com/tensorflow/tensorflow/blob/v2.4.1/tensorflow/python/keras/layers/normalization.py#L996) and [MultiHeadAttention](https://github.com/tensorflow/tensorflow/blob/v2.4.1/tensorflow/python/keras/layers/multi_head_attention.py) are missing from the current implementation.

**Comment by GeorgeS2019** [comment]
@lqdev  In your approach to MultiHeadAttention, perhaps you could share your feedback to this [thread ](https://github.com/xamarin/TorchSharp/issues/257#issuecomment-845394990)of discussion. 

Perhaps we may find more people interested on this topics for .NET

**Comment by lindadamama** [comment]
missing layerNormalization
However, Torchsharp's APIS are complete
https://github.com/tensorflow/tensorflow/tree/v2.7.0/tensorflow/python/keras/layers/normalization
https://github.com/dotnet/TorchSharp/tree/main/src/TorchSharp/NN/Normalization
